bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "legislative-simulation",
+  "version": "0.0.0",
+  "homepage": "https://github.com/jtoney1/legislative-simulation",
+  "authors": [
+    "james <jatoney@hotmail.com>"
+  ],
+  "license": "MIT",
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
```

Comment:
We need to have angular and angular-route in here. On that note, we need to figure out which browsers we need to support in order to know which version of angular to use. Angular 1.3 dropped support for ie8